workflows/AcceptParameter.cwl

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env cwl-runner
+cwlVersion: v1.2
+class: Workflow
+
+label: AcceptAndArchive
+doc: |-
+    Accept and archive simulation model parameter(s).
+    Acceptances includes review of validation process.
+
+inputs:
+
+    - id: model_parameter
+      type: File
+      doc: |-
+        New or updated simulation parameter(s).
+
+    - id: validation_report
+      type: File
+      doc: |-
+        Report on validation process.
+
+outputs:
+
+  - id: acceptance_statement
+    doc: |-
+        Acceptence statement issued by the simulation
+        system or by manual inspection.
+    type: string
+    outputSource: review_validation/acceptance_statement
```

Comment:
OK we can discuss it elsewhere to help me understand the idea here but for now we can move on.

I will mention one annoying point though, you use camelCase everywhere here while we decided to change to snake_case in `simtools`. Maybe it's standard in cwl, but if not, maybe we should consider sticking to snake_case also here.

Reply:
Commands, like `outputSource` in CWL are camelCase and we cannot change.

I have named (historically) workflows with CamelCases and tools / methods with snake_case. This is a very easy way to disentangle both. I think it would be nice to keep this, but I don't have a strong opinion (tools / methods are anyway in the ./tools directory)

Reply:
OK, if that's the way CWL defines those commands then it is fine of course. The other decisions about tools/methods are good (we discussed them before I think).